GetDKAN/dkan2
# Add additional field options for indexing

This branch should work with https://github.com/GetDKAN/data-catalog-frontend/pull/16
Related ticket: https://github.com/getdkan/dkan2/issues/344

## QA Steps
1. Confirm you can add additional schema fields like publisher.name and distribution.title to the search index `/admin/config/search/search-api/index/dkan/fields`
